apps/pay/components/payment-outcome/receipt.tsx

```diff
@@ -61,15 +68,16 @@ function receipt(props: Props) {
             </tr>
           </table>
         </div>
-        <a
-          className={styles.link}
-          href="https://galoy.io"
-          target="_blank"
-          rel="noreferrer"
-        >
-          {" "}
-          Powered by <GaloyIcon />
-        </a>
+        <div className="link-wrapper">
+          <a
+            className={styles.link}
+            href="https://galoy.io"
+            target="_blank"
+            rel="noreferrer"
+          >
+            Powered by <GaloyIcon />
+          </a>
+        </div>
```

Comment:
we can remove that. at least as it is currently, it's taking too much real estate for a POS screen (we should not have the ability to scroll on such a screen really, the focus should be on inputing an amount with a keyboard and we should not distract from this action by making the page longer)

Reply:
This is part receipt PDF. Not for webpage.